src/errors.js

```diff
@@ -95,3 +95,16 @@ export function onlyErrorsWithCode(
     return errorHandler(error);
   };
 }
+
+export function isErrorWithCode(
```

Comment:
I was thinking about it too:
my only concern is that it can be still useful if in the near future we need to catch errors into a promise chain and we don't want (or maybe we can't) use the async functions directly for some reasons that maybe we don't see right now.